.github/workflows/build.yml

```diff
@@ -24,9 +24,28 @@ jobs:
         uses: apache/skywalking-eyes@cd7b195c51fd3d6ad52afceb760719ddc6b3ee91
 
   build:
+    strategy:
+      # amd64,arm64,ppc64le,s390x
+      fail-fast: false
```

Comment:
Does fail-fast default to true?

Reply:
Yes. If one of the targets fail, the rest are cancelled. This allows us to continue with the other archs if one failed. Was definitely helpful when working on this. I think we still want it?